tck/pom.xml

```diff
@@ -23,7 +23,7 @@
 	<parent>
 		<groupId>jakarta.enterprise.concurrent</groupId>
 		<artifactId>jakarta.enterprise.concurrent.parent</artifactId>
-		<version>3.0.0-SNAPSHOT</version>
+		<version>3.0.0</version>
```

Comment:
The primary change in this pull looks good, but I'd recommend not including the version update because it will collide with Steve's #220

Reply:
This was remainder as I was splitting updates to separate branches and PRs. Thanks for spotting!